config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+Kaminari.configure do |config|
+  config.default_per_page = 10
```

Comment:
Please move this value as environment variable

Reply:
Done. https://github.com/BaritoLog/BaritoMarket/pull/134/commits/9a39ff7f03b901ed75307d63abe945ab62079347